src/Microsoft.DotNet.Build.Tasks/GenerateEncodingTable.cs

```diff
@@ -0,0 +1,395 @@
+﻿using Microsoft.Build.Framework;
```

Comment:
I mean anyone doing clean build (like msbuild /t:rebuild) will regenerate the file. resources are changing frequently so this is why we are regenerating it on the fly but this encoding data tables is not expected to change that frequently.